src/toil/jobStores/aws/jobStore.py

```diff
@@ -299,6 +339,73 @@ def deleteFile(self, jobStoreFileID):
         else:
             info.delete()
 
+    nativeUrlRegex = re.compile(r"""
+        # url scheme
+        [sS]3://
+
+        # bucket name
+        #   (1) Must have length 3-255
+        #   (2) Contain only letters, numbers, dashes(-), periods(.), and underscores(_)
```

Comment:
I don't think this is true, by default our toil buckets are appended with "--files" and "--stats"